app/javascript/src/modules/thumbnail.js

```diff
@@ -19,7 +19,7 @@ export default class {
 
     let thumbnailIcon = ''
     if (this.thumbnailUrl) {
-      thumbnailIcon = `<img class="square-icon" src="${this.thumbnailUrl}" alt="" />`
+      thumbnailIcon = `<img data-controller="image" data-action="error->image#handleError auth-success@window->image#updateImage" class="square-icon" src="${this.thumbnailUrl}" alt="" />`
```

Comment:
What is the error we are trapping for here?  Is it possible this catches some other error that we don't want?

Reply:
I don't think so. This should be something default to browser that gets triggered when an image fails to load. http://www.w3schools.com/Jsref/event_onerror.asp

Reply:
Oh, right it's scoped to this element only.